docs/sdk/storage/fragments/android/getting-started.md

```diff
@@ -14,10 +14,10 @@ If you had previously enabled user sign-in by running `amplify add auth` in your
 * Protected: Readable by all users, but writable only by the creating user. Files are stored under `protected/{user_identity_id}/` where the `user_identity_id` corresponds to the unique Amazon Cognito Identity ID for that user.
 * Private: Only accessible for the individual user. Files are stored under `private/{user_identity_id}/` where the `user_identity_id` corresponds to the unique Amazon Cognito Identity ID for that user.
 
-See [Authentication](./authentication) for more information on how to get the `user_identity_id` for a signed in user.
+See [Authentication](~/sdk/auth/getting-started.md) for more information on how to get the `user_identity_id` for a signed in user.
 ## Set Up Your Backend
 
-1. Complete the [Get Started](start) steps before you proceed.
+1. Complete the [Get Started](~/start/start.md) steps before you proceed.
```

Comment:
we should probably point to the Android getting started guide here

Reply:
In my testing, I got this for free without being explicit. See https://pr-1443.d39agkzf0cwtdz.amplifyapp.com/sdk/storage/getting-started?platform=android.